src/index.ts

```diff
@@ -0,0 +1,28 @@
+import express, { NextFunction, Request, Response } from 'express';
+import compression from 'compression';
+import { TEST_CONTROLLER } from './controllers/test-controller';
+import { Server } from 'http';
+
+const APP: express.Application = express();
```

Comment:
I'm don't really like this uppercase style. what do you guys thinks? @hrehman-sphereon and @nklomp

Reply:
Actually I think I went too far with the upper case style for constants. Upper case is only for **global constant values and enum values**. So, I think it's reasonable to use them only for constants that are exported. Reference: https://google.github.io/styleguide/tsguide.html#identifiers.